src/motion/functionality/position.ts

```diff
@@ -101,12 +101,11 @@ export const Position: FunctionalComponentDefinition = {
             values,
             positionTransition,
         }: FunctionalProps) => {
-            return usePositionAnimation(
-                innerRef,
-                values,
-                controls,
-                positionTransition
-            )
+            if (typeof window === "undefined") {
```

Comment:
This check might live better on line 96?

```
shouldRender: (props: MotionProps) => !!props.positionTransition && typeof window !== "undefined",
```

?

Reply:
good point, changed